modules/storages/db/migrate/20230802085026_rename_projects_storages_table.rb

```diff
@@ -0,0 +1,39 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2023 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+class RenameProjectsStoragesTable < ActiveRecord::Migration[7.0]
+  def up
+    rename_table :projects_storages, :project_storages
+    rename_column :last_project_folders, :projects_storage_id, :project_storage_id
+  end
+
+  def down
+    rename_table :project_storages, :projects_storages
+    rename_column :last_project_folders, :project_storage_id, :projects_storage_id
+  end
```

Comment:
```suggestion
  def change
    rename_table :projects_storages, :project_storages
    rename_column :last_project_folders, :projects_storage_id, :project_storage_id
  end
```

We actually shouldn't need to define both as the `change` method can cater for bot up & down

Tested locally

![Screenshot 2023-08-03 at 11 21 43 AM](https://github.com/opf/openproject/assets/17295175/2cd8dae2-5bb4-4a17-80d4-66c3e09dc951)
![Screenshot 2023-08-03 at 11 22 00 AM](https://github.com/opf/openproject/assets/17295175/6ac38849-4740-49b2-bded-9de4e9de779e)

Reply:
I somehow thought the same.